ios/samples/build_common.sh

```diff
@@ -1,56 +1,44 @@
 #!/usr/bin/env bash
 
-# set -e: Terminate script if a command returns an error
-set -e
-# set -u: Terminate script if an unset variable is used
-set -u
-# set -x: Debugging use, print each statement
-# set -x
-
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
-. "${THIS_SCRIPT%/*}/../../resources/build/build-utils.sh"
+. "${THIS_SCRIPT%/*/*}/../resources/build/build-utils.sh"
 ## END STANDARD BUILD SCRIPT INCLUDE
 
+# TODO:  resolve how this is sourced by scripts in subfolders;
+# we want to cd to the subfolder including this script.
+#
+# # This script runs from its own folder
+# cd "$(dirname "$THIS_SCRIPT")"
+
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
```

Comment:
I think we want to setup build_common.sh to be common.inc.sh, with functions that are called from the child scripts, even if it is just one function call that does basically everything.

Reply:
Or would build_common.sh be split into separate builder scripts for KMSample1 and KMSample2?